.github/workflows/publish-sphinx.yml

```diff
@@ -94,15 +94,15 @@ jobs:
         wget https://github.com/nexusformat/code/raw/master/applications/NXtranslate/docs/NXtranslate.pdf -O ./build/manual/source/_static/NXtranslate.pdf
 
     - name: Build (html) and Commit
-      uses: sphinx-notes/pages@master
+      uses: sphinx-notes/pages@main
```

Comment:
Not seeing a branch named `main` here: https://github.com/sphinx-notes/pages/branches